audformat/core/testing.py

```diff
@@ -150,7 +150,6 @@ def add_table(
 
 def create_audio_files(
         db: Database,
-        root: str,
```

Comment:
This is of course a breaking change. The question is - should we officially deprecate it. But since it is a testing function it might be ok to simply remove it.

Reply:
Ok, it's now set to `None` by default and a warning is raised if it is set:

```
UserWarning: The argument 'root' is deprecated, 'db.root' will be used.'
```

Reply:
Great, your solution looks good.